iothub/device/src/DeviceClient.cs

```diff
@@ -20,6 +20,15 @@ public sealed class DeviceClient : IDisposable
         /// </summary>
         public const uint DefaultOperationTimeoutInMilliseconds = 4 * 60 * 1000;
 
+        /// <summary>
+        /// The PnP Model Id associated with the device identity
+        /// </summary>
+        public string PnPModelId
```

Comment:
Does anyone else find the casing for this kind of variable odd? Should this just be "PlugAndPlayModelId" for clarity and to avoid this casing nightmare?

Reply:
Or maybe just call it "ModelId" and then clarify in the comments that this is a plug and play feature?

Reply:
Ah, I thought I did change it to "ModelId" :(

Reply:
Oh, this is in the device client setter; yes, I would like to have the pnp part highlighted, I can change it to "PlugAndPlayModelId" instead.

Reply:
If it was this PnPModeId, I'd lowercase the second P as PnpModelId to follow casing rules.